src/main/java/it/gov/pagopa/ranking/dto/mapper/OnboardingRankingRequest2RankingRequestsApiDTOMapper.java

```diff
@@ -16,7 +16,7 @@ public RankingRequestsApiDTO apply(OnboardingRankingRequests onboardingRankingRe
         out.setAdmissibilityCheckDate(onboardingRankingRequests.getAdmissibilityCheckDate());
         out.setCriteriaConsensusTimestamp(onboardingRankingRequests.getCriteriaConsensusTimestamp());
         out.setRankingValue(onboardingRankingRequests.getRankingValue());
-        out.setRank(onboardingRankingRequests.getRank());
+        out.setRanking(onboardingRankingRequests.getRanking());
 
```

Comment:
fill the new field beneficiaryRankingStatus (it should be added also to OnboardingRankingRequests, and filled in its mapper with TO_NOTIFY)